ui-admin/src/portal/siteContent/HtmlPageEditView.tsx

```diff
@@ -33,6 +37,51 @@ const HtmlPageView = ({ htmlPage, readOnly, siteInvalid, setSiteInvalid, updateP
     updatePage(htmlPage)
   }
 
+  const removeSection = (sectionIndex: number) => () => {
```

Comment:
I should have put these in the page editor to begin with. Sections don't need to care about their index or the overall structure of the page that they belong to. This also made it a lot cleaner to deal with footers which are sections that don't live within an `HtmlPage`.